### PR TITLE
DDF-2875 Unit tests for LdapClaimsHandler

### PR DESCRIPTION
**Pull request by ahoffer**
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
1. Improves unit coverage of LdapClaimsHandler from 28% method coverage to 57% method coverage.
2.Improves unit coverage of LdapClaimsHandler from 11% line coverage to 77% line coverage.
3. Cleans up checkstyle warnings about ClaimsHandlerManager

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
@clockard

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@clockard
@coyotesqrl 
#### How should this be tested? (List steps with links to updated documentation)
Full build of DDF w/ integration tests
#### Any background context you want to provide?
Part of red team's "war on test coverage"
#### What are the relevant tickets?
[DDF-2875](https://codice.atlassian.net/browse/DDF-2875)
#### Checklist:
- [ N/A] Documentation Updated
- [ N/A] Change Log Updated
- [X ] Update / Add Unit Tests
- [ N/A] Update / Add Integration Tests

### CODE REVIEW

File: platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java [hunk 1]
```diff
@@ -78,8 +78,6 @@
```
Comment (ahoffer) [inline platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java]: Cleaned up some checkstyle warnings in this class. Nothing more.

File: platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java [hunk 2]
```diff
@@ -16,67 +16,166 @@
...
+        Map<String, String> map = new HashMap<>();
+        map.put("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier",
+                ATTRIBUTE_NAME);
+        PowerMockito.mockStatic(AttributeMapLoader.class);
+        when(AttributeMapLoader.buildClaimsMapFile(anyString())).thenReturn(map);
+        when(AttributeMapLoader.getUser(any(Principal.class))).then(i -> i.getArgumentAt(0,
+                Principal.class)
+                .getName());
+
+        //TODO: Test the path where the method getUserBaseDN() returns null.
```
Comment (brendan-hofmann) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java]: Ticket for the todo or are you planning on addressing it in this pr?

Comment (ahoffer) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java]: Fixed.

**Comment by ahoffer** [comment]
build now

File: platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java [hunk 3]
```diff
@@ -16,67 +16,165 @@
...
+    ClaimCollection claims;
+
+    ConnectionEntryReader mockEntryReader;
+
+    SearchResultEntry mockEntry;
+
+    ClaimsParameters claimsParameters;
+
+    @Before
+    public void setup() throws LdapException, URISyntaxException, SearchResultReferenceIOException {
```
Comment (clockard) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java]: Could just throw Exception here

File: platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java [hunk 4]
```diff
@@ -16,67 +16,165 @@
...
+        mockEntryReader = mock(ConnectionEntryReader.class);
+        mockBindRequest = mock(BindRequest.class);
+        PowerMockito.mockStatic(BindMethodChooser.class);
+        when(BindMethodChooser.selectBindMethod(eq(BINDING_TYPE),
+                eq(BIND_USER_DN),
+                eq(BIND_USER_CREDENTIALS),
+                eq(REALM),
+                eq(KCD))).thenReturn(mockBindRequest);
+        Map<String, String> map = new HashMap<>();
+        map.put("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier",
```
Comment (clockard) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/LdapClaimsHandlerTest.java]: There is a constant defined somewhere for this.

**Comment by ahoffer** [comment]
build now

**Pull request merged**